okhttp3.astub

```diff
@@ -0,0 +1,71 @@
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.dataflow.qual.SideEffectFree;
+import org.checkerframework.common.returnsreceiver.qual.This;
+import org.checkerframework.framework.qual.*;
+
+package okhttp3;
+
+
+class Response{
+    @Owning ResponseBody body;
+
+
+    @MustCallAlias Request request(@MustCallAlias Response this);
+    @MustCallAlias ResponseBody body(@MustCallAlias Response this);
+    @MustCallAlias Response.Builder newBuilder(@MustCallAlias Response this); //This is used to connected the shallow-copy created to the Builder Constructor.
```

Comment:
I would have expected this to be `@MustCall({})` to match the `Builder` constructor directly. Does this work?

I'm also not sure why the `Response` on which `newBuilder()` is called should have anything to do with whether the new `Builder` should have must-call obligations - shouldn't it be fresh?